### PR TITLE
hard link tea binary into default PATH

### PR DESCRIPTION
**Pull request by mxcl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,12 @@
...
+A container with `tea` and its magic installed and functional.
 
 ```sh
-docker buildx build \
-  --pull --push \
-  --tag ghcr.io/teaxyz/infuser:latest \
-  --platform linux/amd64,linux/arm64 \
-  --file infuser/Dockerfile \
-  .
+docker run --rm -it ghcr.io/teaxyz/infuser /bin/bash
```
Comment (jhheider) [inline README.md:6]: Can remove `/bin/bash` if you set an `ENTRYPOINT` in the `Dockerfile`.

Comment (mxcl) [reply README.md:6]: oh neat

Comment (mxcl) [reply README.md:6]: updated

Comment (mxcl) [reply README.md:6]: This apparently makes it impossible to do anything else with the container, like run a script… unless I'm doing it wrong.

Comment (mxcl) [reply README.md:6]: oh ok you can override with `--entrypoint`

Comment (jhheider) [reply README.md:6]: Hm. I was pretty sure that if you added a command it overrode `ENTRYPOINT` by default.

Ah, it's `CMD` that is auto-override. https://docs.docker.com/engine/reference/builder/#cmd

**Pull request merged**